ansible_collections/arista/avd/roles/eos_l3ls_evpn/tasks/main.yml

```diff
@@ -32,10 +40,46 @@
   tags: [build, provision]
 
 - name: Generate Fabric Topology in csv format.
+  when: fabric_name is defined
   template:
     src: documentation/dc-fabric-topology.j2
     dest: '{{ root_dir }}/documentation/{{ fabric_name }}/{{ fabric_name }}-topology.csv'
   delegate_to: localhost
   run_once: true
   check_mode: no
   tags: [build, provision]
+
+- name: Create Directory for the dc topology documentaion
+  file:
+    path: '{{ root_dir }}/documentation/{{ dc_name }}'
+    state: directory
+
+- name: Generate DC Topology in csv format.
```

Comment:
I would suggest that we don't have dedicated tasks for 5 stage clos fabric scenario and rather handle this logic in the templates.